src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/UseNameofInPlaceOfString.Fixer.cs

```diff
@@ -47,15 +46,15 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 context.Diagnostics);
         }
 
-        private async Task<Document> ReplaceWithNameOf(Document document, SyntaxNode nodeToReplace,
+        private static async Task<Document> ReplaceWithNameOf(Document document, SyntaxNode nodeToReplace,
             string stringText, CancellationToken cancellationToken)
         {
             var editor = await DocumentEditor.CreateAsync(document, cancellationToken).ConfigureAwait(false);
             var generator = editor.Generator;
 
             var trailingTrivia = nodeToReplace.GetTrailingTrivia();
             var leadingTrivia = nodeToReplace.GetLeadingTrivia();
-            SyntaxNode nameOfExpression = GetNameOfExpression(generator, stringText)
+            var nameOfExpression = generator.NameOfExpression(generator.IdentifierName(stringText))
```

Comment:
The usage of explicit type generates an IDE0007 warning locally. I'm completely confused how this doesn't fail CI?

Reply:
Actually, there are **tons** of warnings locally. Including RS1024 which is explicitly disabled in editorconfig

![image](https://user-images.githubusercontent.com/31348972/105221940-7737cb80-5b62-11eb-9195-32208c6db09b.png)

![image](https://user-images.githubusercontent.com/31348972/105222034-96365d80-5b62-11eb-9cb5-49bfc6ea75d4.png)